MapleServer2/PacketHandlers/Game/RequestItemUseHandler.cs

```diff
@@ -0,0 +1,51 @@
+using Microsoft.Extensions.Logging;
+using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Servers.Game;
+using MapleServer2.Packets;
+using MapleServer2.Types;
+
+namespace MapleServer2.PacketHandlers.Game
+{
+    public class RequestItemUseHandler : GamePacketHandler
+    {
+        public override RecvOp OpCode => RecvOp.REQUEST_ITEM_USE;
+
+        public RequestItemUseHandler(ILogger<RequestItemUseHandler> logger) : base(logger) { }
+
+        public override void Handle(GameSession session, PacketReader packet)
+        {
+            long boxUid = packet.ReadLong();
+
+            if (!session.Player.Inventory.Items.ContainsKey(boxUid))
+            {
+                return;
+            }
+
+            // Get the box item
+            Item box = session.Player.Inventory.Items[boxUid];
+
+            // Normally would look up which item to create, instead always create poisonous mushroom
+            Item item = new Item(30001001)
```

Comment:
This should probably live around line 47 since that's the first time it is being referenced